great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -90,6 +92,12 @@
 try:
     import sqlalchemy as sa
 
+    if version.Version(sa.__version__) >= version.Version("2.0.0"):
```

Comment:
can you please move this into a utility method (we already import a number of them) so that this code isn't duplicated.

`sql_alchemy_version_check(sa.__version__)`